spec/annotator-content-analysis_spec.js

```diff
@@ -1,6 +1,33 @@
+const options = {
+  dataUrl: '../demo/tua.json',
+  templates: {
+    form: '/templates/form.handlebars',
+    topic: '/templates/topic.handlebars',
+    question: '/templates/question.handlebars',
+    answer: '/templates/answer.handlebars'
+  }
+};
+
+
+
 describe('Annotator Custom Editor ', function() {
-  return describe('On loading the module', function() {
-    return it('should set the data', function() {
+  const analyzer = annotatorContentAnalysis(options);
+  analyzer.start();
```

Comment:
if you reference annotator-content-analysis_demo.js you'll see that we include our module into annotator core and then start that. for more reference i'd check out how annotator itself is writing its own tests: https://github.com/openannotation/annotator/blob/master/test/spec/app_spec.js